.github/ISSUE_TEMPLATE/bug-issue-template.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug report
+description: Bug report template.
+title: '[Bug]: '
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: 'Before you open an issue, please verify if a similar one already exists or has been closed before. More details about the process of contributing can be found in [CONTRIBUTING.md](https://github.com/TheAlgorithms/JavaScript/blob/master/CONTRIBUTING.md).'
+  - type: textarea
```

Comment:
We should maybe add a `Description` field here for the user to explain their problem.